spec_util/ir_hash/gen/gen.go

```diff
@@ -29,6 +29,7 @@ func main() {
 	gf.SetPackageName("ir_hash")
 	gf.AddImports()
 
+	gf.AddIgnoredField("HTTPMethodMeta", "PathParamAlg")
```

Comment:
Hm, this is an argument I'd forgotten about not putting things in the IR -- we want to be able to hash it to check for equality.  (Which is one reason I regret putting ProcessingLatency here.)

I think not including it in the hash is correct, though. Two methods with identical content but different PathParamAlg is correct.